src/components/RoomDialog.tsx

```diff
@@ -0,0 +1,168 @@
+
+import "./RoomDialog.scss";
+import { copyTextToSystemClipboard } from "../clipboard";
+
+function RoomModal({
+  onCloseRequest,
+  activeRoomLink,
+  onStartSession,
+}: {
+  onCloseRequest: () => void;
+  activeRoomLink: string;
+  onStartSession: () => void;
+}) {
+  const roomLinkInput = useRef<HTMLInputElement>(null);
+  function copyRoomLink() {
+    copyTextToSystemClipboard(activeRoomLink);
+    if (roomLinkInput.current) {
+      roomLinkInput.current.select();
+    }
+  }
+  function selectInput(event: React.MouseEvent<HTMLInputElement>) {
+    if (event.target !== document.activeElement) {
+      event.preventDefault();
+      (event.target as HTMLInputElement).select();
+    }
+  }
+
+  return (
+    <div className="RoomDialog-modal">
+      <Island padding={4}>
+        <button
+          className="Modal__close"
+          onClick={onCloseRequest}
+          aria-label={t("buttons.close")}
+        >
+          ╳
+        </button>
+        <h2 id="export-title">{t("labels.createRoom")}</h2>
+        {!activeRoomLink && (
+          <>
+            <p>{t("roomDialog.desc_intro")}</p>
+            <p>{t("roomDialog.desc_privacy")}</p>
+            <p>{t("roomDialog.desc_start")}</p>
+            <div className="RoomDialog-sessionStartButtonContainer">
+              <ToolButton
+                className="RoomDialog-startSession"
+                type="button"
+                icon={start}
+                title={t("roomDialog.button_startSession")}
+                aria-label={t("roomDialog.button_startSession")}
+                showAriaLabel={true}
+                onClick={onStartSession}
+              />
+            </div>
+          </>
+        )}
+        {activeRoomLink && (
+          <>
+            <p>{t("roomDialog.desc_inProgressIntro")}</p>
+            <p>{t("roomDialog.desc_shareLink")}</p>
+            <div className="RoomDialog-linkContainer">
+              <ToolButton
+                type="button"
+                icon={clipboard}
+                title={t("labels.copy")}
+                aria-label={t("labels.copy")}
+                onClick={copyRoomLink}
+              />
+              <input
+                value={activeRoomLink}
+                readOnly={true}
+                className="RoomDialog-link"
+                ref={roomLinkInput}
+                onPointerDown={selectInput}
+              />
+            </div>
+            <p>{t("roomDialog.desc_privacy")}</p>
+            <p>{t("roomDialog.desc_exitSession")}</p>
+            <p>{t("roomDialog.desc_createAnotherSession")}</p>
+            <div className="RoomDialog-sessionStartButtonContainer">
+              <Stack.Col gap={1}>
+                <Stack.Row gap={2}>
+                  <ToolButton
+                    className="RoomDialog-createLink"
+                    type="button"
+                    icon={start}
+                    title={t("roomDialog.button_startAnotherSession")}
```

Comment:
You just started a session, it feels weird to have the call to action be to start another session. I would remove that.

Reply:
Yea, I wasn't sure myself. But if we remove this, you won't be able to create a different session. Probably doesn't matter.